physics/escape_velocity.py

```diff
@@ -0,0 +1,74 @@
+"""
+Title: Calculate Escape Velocity
+
+Description:
+    This algorithm calculates the escape velocity required for an object to
+    leave the gravitational influence of a celestial body.
+"""
+
+import math
+
+
+def calculate_escape_velocity(
```

Comment:
Let's simplify the name.
```suggestion
def escape_velocity(
```
and change other instances in this file.